dxr/static/css/dxr.css

```diff
@@ -198,7 +195,39 @@ mark {
     background-color: transparent;
 }
 
+/* promoted stuff, on the right */
+.promoted-container {
+    /* cannot grow, can shrink, prefers 30% size */
+    flex: 0 1 30%;
+}
```

Comment:
I did a little basic painting so we can get this landed. This doesn't look too bad:
```
border: 1px solic #a0a0a0;
border-radius: 4px;
padding: 1.5em 2em;
```
To make the perceived padding inside the box look symmetrical, we also need to apply this to .promoted-see-more:
```
margin: 0 0 1rem
```
(That also makes it consistent with .top-of-tree, the header for the ordinary results.)

And, finally, we need to change .promoted-row from `margin-bottom :5px` to…
```
margin-top: .4rem
```